packages/@o3r/analytics/tsconfig.build.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../../tsconfig.build",
   "compilerOptions": {
+    "skipLibCheck": true, // Due to Jasmine / Jest fixture compilation conflicting on types
```

Comment:
Could it be useful to have the list of libs causing this ? (@o3r/core, @o3r/schematics etc )

Reply:
It is actually all the modules with fixtures on Jasmine and Jest.
Where would you like to have this list? (And for which purpose? Not sure it is a argument enough strong to move out from Jasmine or Fixture mechanism :S)

Reply:
I see. I was thinking to have the list of the packages which have caused the activation of the flag, here in the comment. But it's true that it can be found easily looking at the deps in package.json.